php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S4784.html

```diff
@@ -75,4 +75,6 @@ <h2>See</h2>
   </li>
   <li> OWASP Regular expression Denial of Service - ReDoS </li>
 </ul>
+<h2>Deprecated</h2>
+<p>This rule is deprecated; use {rule:php:S2631} instead.</p>
```

Comment:
SonarQube displays that as a link, but it doesn't work. It should be `{rule:phpsecurity:S2631}` instead of `{rule:php:S2631}`. The problem is that phpsecurity:S2631 is not available in SQ Community Edition... Maybe we can still keep the link.